k8-operator/controllers/infisicalpushsecret/infisicalpushsecret_controller.go

```
@@ -65,6 +65,8 @@ func (r *InfisicalPushSecretReconciler) Reconcile(ctx context.Context, req ctrl.
 	if err != nil {
 		if errors.IsNotFound(err) {
 			logger.Info("Infisical Push Secret CRD not found")
+			r.DeleteManagedSecrets(ctx, logger, infisicalPushSecretCRD)
```

golangci-lint (1.64.8)

Error return value of `r.DeleteManagedSecrets` is not checked

(errcheck)